tests/component.test.js

```diff
@@ -0,0 +1,163 @@
+const ParchKey = require('../model/parch-key.js');
+const Component = require('../model/component.js');
+const Port = require('../model/port.js');
+const Coord = require('../model/coord.js');
+
+//Suppress console logs
+console.log = jest.fn();
+
+var l1 = 'layer-1-id';
+var l2 = 'layer-2-id';
+var p1 = new Port('port-1-label', l1, new Coord(0, 0));
+var p2 = new Port('port-2-label', l2, new Coord(0, 5));
+
+test('initialize Component: parameters', () => {
+    let cVal = new Component('name', 'id', [l1, l2], 10, 10, 'entity', [new Port(), new Port()]);
```

Comment:
Why not use `p1` and `p2` here?

Reply:
Probably because I wrote that test before creating those globals. It uses them now.